pkg/results/types.go

```diff
@@ -50,26 +60,30 @@ type ScanSummary struct {
 	IacType              string `json:"iac_type" yaml:"iac_type" xml:"iac_type,attr"`
 	Timestamp            string `json:"scanned_at" yaml:"scanned_at" xml:"scanned_at,attr"`
 	ShowViolationDetails bool   `json:"-" yaml:"-" xml:"-"`
-	TotalPolicies        int    `json:"policies_validated" yaml:"policies_validated" xml:"policies_validated,attr"`
-	ViolatedPolicies     int    `json:"violated_policies" yaml:"violated_policies" xml:"violated_policies,attr"`
+	RulesValidated       int    `json:"rules_validated" yaml:"rules_validated" xml:"rules_validated,attr"`
```

Comment:
The work `RuleValidated` gives the same sound as rules approved. Could we go with Rules Tried?

Reply:
May be `Evaluated`?